server/services/shortenedUrlRepository.js

```diff
@@ -33,7 +33,7 @@ class ShortenedUrlsRepository extends Array{
     }
 
     getAll(){
-        return this;
+        return Object.keys(this);
```

Comment:
This class can be deleted.

Reply:
According to the document we need to only return a list of the used keys. This would return more, would it not? I can double check though

Reply:
Correct. But you're looking at the old repository. I dont use this class anymore. Check urlRepository.js. There I return a list of keys.

Reply:
🙈